pkg/controller/cluster/operator.go

```diff
@@ -202,9 +202,9 @@ func (c *Controller) createOperatorSecret(ctx context.Context, operator metav1.O
 			Labels:    labels,
 			OwnerReferences: []metav1.OwnerReference{
 				*metav1.NewControllerRef(operator, schema.GroupVersionKind{
-					Group:   miniov2.SchemeGroupVersion.Group,
-					Version: miniov2.SchemeGroupVersion.Version,
-					Kind:    miniov2.OperatorCRDResourceKind,
+					Group:   "apps",
+					Version: "v1",
+					Kind:    "Deployment",
 				}),
 			},
```

Comment:
is it good to hardcode these values? what are the implications?

Reply:
not much really, we are passing a Deployment object to this function, but it's behind the `metav1.Object` interface

Reply:
Make sense, theres no operator CRD, but how does it know we want to reference the operator deployment?

Reply:
@Alevsk the `operator` reference can only be a deployment, following the code logic